src/Jira/Api.php

```diff
@@ -198,6 +198,16 @@ public function editIssue($issue_key, array $params)
 	}
 
 	/**
+	 * Gets attachment upload limit.
+	 *
+	 * @return array
+	 */
+	public function getAttachmentUploadLimit()
```

Comment:
Please rename into `getAttachmentsMeta`, because according to https://docs.atlassian.com/jira/REST/latest/#api/2/attachment-getAttachmentMeta not only upload limit is returned but all attachment configuration set in given JIRA instance.